CHANGELOG.md

```diff
@@ -15,13 +17,31 @@ NOTE: In the remote chance that the `system.workflow-heartbeats.ttl` has been co
 then the new configuration value `system.workflow-heartbeats.write-failure-shutdown-duration` must also be explicitly
 set less than the `ttl`.
 
+#### Logging long running jobs
+
+All backends can now emit slow job warnings after a configurable time running. 
+NB This example shows how to configure this setting for the PAPIv2 backend:
+```conf
+# Emit a warning if jobs last longer than this amount of time. This might indicate that something got stuck.
+backend {
+  providers {
+    PAPIv2 {
+      config { 
+        slow-job-warning-time: 24 hours
+      }
+    }
+  }
+}
+```
+
 ### Bug fixes
 
 #### Better validation of workflow heartbeats
 
 An error will be thrown on startup when the `system.workflow-heartbeats.heartbeat-interval` is not less than the
 `system.workflow-heartbeats.ttl`.
 
+
```

Comment:
ToL: If reverting the other whitespace change, I'd revert this also. The CHANGELOG is already susceptible to merge-conflicts, especially in this `Bug fixes` region.